examples/performance/laplace/data/BOUT.inp

```diff
@@ -0,0 +1,24 @@
+# Test of the FieldFactory class 
+#
+# Generates a set of fields and writes them to output file
+#
+
+NOUT = 0  # No timesteps
+
+MZ = 32    # Z size
+
+grid = "../../../tests/integrated/test-laplace/test_laplace.grd.nc"
```

Comment:
I think the information in these grid files is pretty trivial. Maybe better to put the variables into `[mesh]` section here? Might even just be defaults `dx=dy=dz=g11=g22=g33=1, g12=g13=g23=0` at a guess.

Reply:
I checked the gridfile, and really all that's in it that's non-default is `nx` and `ny`, so can replace with
```
nx = 12
ny = 16
```
in the `[mesh]` section. Actually for a performance test, probably want to increase those numbers a bit... I guess `ny` does not really matter, effectively it just multiplies the number of iterations and averages over a few slightly different inputs, but a `nx` and `mz` in the range 100-200 is probably more representative.

Reply:
I think this is following the style of the other performance tests/examples where a scan over grid size is one of the use cases (to look at weak scaling as well as strong scaling etc.). In the other performance cases I think this is included as a part of the script used to run the scan.

Reply:
Thanks both. I was following the `iterator/make_scaling_example.sh` style in `runtest`, but I stripped out the grid size scan.  I can put that back in if people prefer? 

(Also, I went with small numbers just to have something that runs in a few seconds on O(1) cores by default.)

Reply:
If you wanted to be fancy you could add some arguments to the script so you can ask for a grid size scan, proc scan or both.